src/style.css

```
@@ -1 +1,37 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
```

you can use `:not(:last-child)` instead
```suggestion
.stars__star {
  width: 16px;
  height: 16px;
  background-image: url(images/star.svg);
  background-position: center;
  background-repeat: no-repeat;
}

.stars__star:not(:last-child) {
  margin-right: 4px;
}
```